.well-known/security.txt

```diff
@@ -1,7 +1,6 @@
 Contact: mailto:security@login.gov
+Contact: https://hackerone.com/gsa_bbp
 Contact: https://login.gov/contact/
-Contact: https://docs.google.com/forms/d/e/1FAIpQLScuo4xCzBlpLnoq7-bDAVAxtJci03by7S-Q-Z_JUBDloK01QA/viewform
 Preferred-Languages: en
 Canonical: https://login.gov/.well-known/security.txt
 Policy: https://handbook.tts.gsa.gov/general-information-and-resources/tech-policies/responding-to-public-disclosure-vulnerabilities/
-Hiring: https://join.tts.gsa.gov/join/security-ops-engineer/
```

Comment:
Worth keeping and pointing to the root page?

https://join.tts.gsa.gov/

Reply:
+1 I would be in favor of pointing to TTS

Reply:
Updated!